### ISSUE TITLE
Discrete Screen - 

### ISSUE THREAD

**Issue by arouinfar** [phetsims/fourier-making-waves#160]
Steps to reproduce
1. Discrete screen
2. Select Wave Packet waveform
3. Zoom in on the x-axis to [-0.25, 0.25]
4. Select function of space and time
5. Notice the shifting autoscale in the Su

**Comment by arouinfar** [comment]
Oops, accidentally submitted an issue while still writing it up. Please disregard until I edit it.

**Comment by pixelzoom** [comment]
Good catch.  It's changing scale because the wave packet's peak is (briefly) not visible on the chart. 

The [-0.25, 0.25] x-axis zoom level has been deleted in the above commit.

@arouinfar please review in master.
